version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "0.0.1-SNAPSHOT"
+ThisBuild / version := "0.0.2-SNAPSHOT"
```

Comment:
This shouldn't need manually bumping. Will need to look at the build action to see what the problem is with it.

Reply:
Looks like the issue is with the `Version bump` label. I have added it manually. Hopefully, in the next deployment, it will work.